src/MergeContext.js

```diff
@@ -536,8 +536,6 @@ class CommitMessage
         // overwrites this._defaultAuthor's name and email
         this._customAuthor = null;
 
-        this._checkRaw(this._title);
```

Comment:
The old check was also a little flawed, because `ths._title` is not 'raw' but already edited title... This makes me think that our 72 character check is misplaced - this limitation should be applied to the output message (i.e., CommitMessage.whole()), not to the input/raw one. In other words, we can validate the result with certainty only after all N editing steps have been applied and the final message is formed.

Reply:
I do not disagree, but I did not want to bring that up in the PR about whitespace trimming. We have another Jira issue dedicated to the length check improvements... Here, we just need to avoid breaking things as far as the 72-character limit testing is concerned.